en/docs/install-and-setup/configuring-api-manager-for-ob.md

```diff
@@ -131,7 +131,130 @@ account retrieval. By default, this is disabled and the configuration is set to
     Enabled = true
     ```
     
-12. If you want to use the [Data publishing](../learn/data-publishing.md) feature:
+12. To validate API requests:
+
+    !!! note
+        This is only available as a WSO2 Update from **WSO2 Open Banking API Manager UK Toolkit Level 1.0.0.6** and
+        **WSO2 Open Banking Identity Server UK Toolkit Level 1.0.0.7** onwards. For more information on updating, see
+        [Getting WSO2 Updates](setting-up-servers.md#getting-wso2-updates).
+
+    ??? tip "Click here to see JWS Signature Validation configurations"
+
+         1. Configure the `UKJwsRequestHandlingExecutor` executor. The priority of `UKJwsRequestHandlingExecutor` 
+            must be higher than `ConsentEnforcementExecutor`. For example:
+           ``` toml 
+           [[open_banking.gateway.openbanking_gateway_executors.type.executors]]
+           name = "com.wso2.openbanking.uk.gateway.executors.jws.UKJwsRequestHandlingExecutor"
+           priority = 4
+
+           [[open_banking.gateway.openbanking_gateway_executors.type.executors]]
+           name = "com.wso2.openbanking.uk.gateway.executors.idempotency.UKIdempotencyHandlingExecutor"
+           priority = 5
+
+           [[open_banking.gateway.openbanking_gateway_executors.type.executors]]
+           name = "com.wso2.openbanking.accelerator.gateway.executor.impl.consent.ConsentEnforcementExecutor"
+           priority = 6    
+           ```
+    
+         2. Enable validation and define the valid signing algorithms for the JWS sent in the request header:
+           ``` toml 
+           [open_banking.jws_signature.signature_validation]
+           enabled=true
+        
+           [[open_banking.jws_signature.signature_validation.allowed_algorithms]]
+           algorithm="PS256"
+        
+           [[open_banking.jws_signature.signature_validation.allowed_algorithms]]
+           algorithm="ES256"
+           ```
+ 
+         3. The signing keys used for validation by an application are cached. The default expiration time for cache 
+            modification and access are 60 minutes. To change these values, add and configure the following:
+           ``` toml
+           [open_banking.common.identity.cache]
+           cache_modified_expiry_minuites=30
+           cache_access_expiry_minuites=30
+           ```
+
+         4. The default trust anchor used for the validation is `openbanking.org.uk`. To change this value, 
+            add and configure the following:
+           ``` toml
+           [open_banking.uk.jws_signature.obie]
+           org_id="0015800001HQQrZAAX"
+        
+           [open_banking.uk.jws_signature.obie.trusted_anchors]
+           signature_validation="openbanking.org.uk"
+           ```
+
+         5. Configure the API contexts that require signature validation:
+          ``` toml
```

Comment:
shall we mention about payments API. validation is enabled by default for payments api

Reply:
Fixed in 0913f39d3e68e6d5c0f106f6490a307c24c16758